src/components/inputs/localized-rich-text-input/editor.js

```diff
@@ -0,0 +1,240 @@
+  const onToggle = React.useCallback(() => {
+    toggleLanguage(props.language);
+  }, [toggleLanguage, props.language]);
+
+  const updateRenderToggleButton = React.useCallback(() => {
+    const doesExceedCollapsedHeightLimit =
+      ref.current.clientHeight > COLLAPSED_HEIGHT;
+
+    if (doesExceedCollapsedHeightLimit && !renderToggleButton) {
+      setRenderToggleButton(true);
+    }
+    if (!doesExceedCollapsedHeightLimit && renderToggleButton) {
+      setRenderToggleButton(false);
+    }
+  }, [setRenderToggleButton, renderToggleButton]);
+
+  React.useEffect(() => {
+    updateRenderToggleButton();
+  }, [props.editor.value.document, updateRenderToggleButton]);
+
+  // opens the input if it regains focus and it's closed
+  if (
+    prevIsFocused !== props.editor.value.selection.isFocused &&
+    props.editor.value.selection.isFocused &&
+    !props.isOpen
+  ) {
+    onToggle();
+  }
+
+  const languagesControl = props.languagesControl();
+
+  return (
+    <CollapsibleMotion
+      minHeight={COLLAPSED_HEIGHT}
+      isClosed={!props.isOpen}
+      onToggle={onToggle}
+      isDefaultClosed={!props.defaultExpandMultilineText}
+    >
+      {({ isOpen, toggle, containerStyles, registerContentNode }) => {
+        return (
+          <Spacings.Stack scale="xs">
+            <div
+              key={props.language}
+              css={css`
+                width: 100%;
+                position: relative;
+                display: flex;
+              `}
+            >
+              <label
+                htmlFor={props.id}
+                css={theme => getLanguageLabelStyles(props, theme)}
+              >
+                {/* FIXME: add proper tone for disabled when tones are refactored */}
+                <Text.Detail tone="secondary">
+                  {props.language.toUpperCase()}
+                </Text.Detail>
+              </label>
+
+              <RichTextBody
+                ref={{
+                  containerRef: ref,
+                  registerContentNode,
+                }}
+                styles={{
+                  container: css`
+                    flex: auto;
+                    width: 100%;
+                    border-top-left-radius: 0;
+                    border-bottom-left-radius: 0;
+                  `,
+                }}
+                isOpen={props.isOpen}
+                hasError={props.hasError}
+                isDisabled={props.isDisabled}
+                hasWarning={props.hasWarning}
+                isReadOnly={props.isReadOnly}
+                editor={props.editor}
+                containerStyles={containerStyles}
+                showExpandIcon={props.showExpandIcon}
+                onClickExpand={props.onClickExpand}
+              >
+                {props.children}
+              </RichTextBody>
+            </div>
+            <Row>
+              {(() => {
+                if (props.error)
+                  return (
+                    <LeftColumn>
```

Comment:
This is similar to what we do in `LocalizedMultilineTextInput`. If the content is empty, we don't want to have extra margin from the `Spacing.Stack`.